setup.py

```diff
@@ -48,7 +49,29 @@ def get_readme() -> str:
         return f.read()
 
 
+class InstallCommand(install):
+
+    def run(self):
+        """Run the install command."""
+        install.run(self)
+        from sys import platform
```

Comment:
Is there a reason the imports are nested? We usually go for top level in line with https://google.github.io/styleguide/pyguide.html#313-imports-formatting

Reply:
My bad, that was Copilot. Will change now.